onnxruntime/core/providers/openvino/openvino_execution_provider.h

```diff
@@ -189,7 +189,9 @@ class OpenVINOExecutionProvider : public IExecutionProvider {
   const void* GetExecutionHandle() const noexcept override {
     return nullptr;
   }
-
+#ifdef USE_OVEP_NPU_MEMORY
+  std::vector<AllocatorPtr> CreatePreferredAllocators() override;
```

Comment:
Not sure how common this would be, but if one uses the perftest tool with the wrong build of OVEP (and provides the `use_device_mem` option), this symbol would not exist.

Perhaps `CreatePreferredAllocators()` should always be defined by OVEP. The implementation could return a default value based on `#ifdef USE_OVEP_NPU_MEMORY`.

Reply:
agree with this.  
can this be addressed in follow-up PR